src/main/resources/templates/subscription.html

```diff
@@ -251,24 +251,32 @@ <h3 class="modal-title text-center" id="formHeader">Subscription Form</h3>
                                     </select>
                                 </div>
                             </div>
-                            <div class="p-1 border form-group" data-bind="visible: authenticationType() == 'BASIC_AUTH'">
+                            <div class="p-1 border form-group" data-bind="visible: authenticationType() == 'BASIC_AUTH' || authenticationType() == 'BASIC_AUTH_JENKINS' ">
```

Comment:
You might be able to use indexOf on an observable. I haven't tried it myself but maybe something like this 'authenticationType().indexOf('BASIC_AUTH') !== -1' would work?